shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt

```diff
@@ -10,25 +11,34 @@ import com.epmedu.animeal.users.domain.model.UserGroup
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
 import javax.inject.Inject
 
 class FeedingsButtonHandlerImpl @Inject constructor(
+    private val buildConfigProvider: BuildConfigProvider,
     private val getUserGroupUseCase: GetCurrentUserGroupUseCase,
     private val getIsNewFeedingPendingUseCase: GetIsNewFeedingPendingUseCase
 ) : FeedingsButtonHandler {
 
     override fun getFeedingsButtonState(): Flow<FeedingsButtonState> {
-        return combine(
-            getIsNewFeedingPendingUseCase(),
-            flow { emit(getUserGroupUseCase()) }
-        ) { isNewFeedingPending, userGroupResult ->
-            when {
-                userGroupResult.isEligibleForFeedingsButton() -> {
-                    if (isNewFeedingPending) Pulsating else Static
-                }
+        return when {
+            buildConfigProvider.isProdFlavor -> {
```

Comment:
This way feedings button would be visible at dev/test environment only

Reply:
Yes, we do not need it in prod until it is fully ready

Reply:
Pls add a tracking ticket for this and add it as comment, we will need to know what changes are necessary in order to make it work on prod env.

Reply:
Done